cmd/prometheus.go

```diff
@@ -43,3 +48,67 @@ func RunPrometheusHTTPHandler(connectionString string) (net.Listener, *http.Serv
 	}()
 	return listener, server, nil
 }
+
+var registerLock = sync.Once{}
+
+// serviceNameToLabelFormat convert service name to lower case and remove all '_'
```

Comment:
IMO, translating `Acra-Server` into `acra_server` is more readable. But I guess the idea behind _removing_ the dashes is to keep the service name as one word so that it's more easily distinguishable in labels?
There's also typo in the comment (`_` → `-`), we replace dashes, not underscores.

```suggestion
// serviceNameToLabelFormat convert service name to lower case and remove all '-'
```

Reply:
@Lagovas is this renaming crucial? 

I would rather not introduce the _third_ type of naming (currently we have `acra-server` as service name, and binary name etc, and `AcraServer` for human-friendly logs and texts).

Reply:
all our metrics have next naming: `acraconnector_*`/`acraserver_*`/`acratranslator_*` according to our discussion in T662
this function was added to automatically convert serviceName to correct metric's format